### ISSUE TITLE
Is it possible to limit the slope of the layers relative to a horizontal stationary bed to make it useful for hobbyist off the shelf printers?

### ISSUE THREAD

**Issue by homasss413** [RyanTaoLiu/NeuralSlicer#2]
I saw prof. Charlie said it is feasible under this video:  https://www.youtube.com/watch?v=LF1fRD7ygdE
Any instructions on this?

**Comment by RyanTaoLiu** [comment]
Yes, we are testing this function now and it will be added to this project.

**Comment by RyanTaoLiu** [comment]
Please feel free to check our new paper, https://dl.acm.org/doi/10.1145/3730922
and the implication of 3Axis https://github.com/RyanTaoLiu/NeuralTOMO/blob/ed5d5e40003377f7612f5612c6f8a4194e242d3b/module/TOuNNOptimizaerFEA.py#L273
